docs/api-usage.rst

```diff
@@ -179,6 +179,17 @@ resources. For example:
    project = gl.projects.get(1)
    project.star()
 
+You can print a Gitlab Object. For example:
+
+.. code-block:: python
+
+   project = gl.projects.get(1)
+   print(project)
+   # Or in a prettier format. These two are equivalent.
+   print(project.pformat())
+   project.pprint()
```

Comment:
Just thinking if we should show them separately a little bit to emphasize it's 2 ways of doing the same thing rather than 2 steps.

```suggestion
   # Print in a prettier format.
   print(project.pformat())
   
   # Or explicitly via `pformat()`. This is equivalent to the above.
   project.pprint()
```

Reply:
Done. Though I flipped `pprint` and `pformat` as I think that is what you meant.